Tests/ttLib/tables/_n_a_m_e_test.py

```diff
@@ -75,6 +75,28 @@ def test_addName(self):
 		with self.assertRaises(TypeError):
 			table.addName(b"abc")  # must be unicode string
 
+	def test_removeNames(self):
```

Comment:
you could also add explicit tests for `removeNames` when called with more than one argument, especially one with all four of them.